<launch>
	<arg name="marker_size" default="14.5" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/rb1_base/front_rgbd_camera/rgb/image_raw" />
	<arg name="cam_info_topic" default="/rb1_base/front_rgbd_camera/rgb/camera_info" />
	<arg name="output_frame" default="rb1_base_front_rgbd_camera_link" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen">
		<param name="marker_size"           type="double" value="$(arg marker_size)" />
		<param name="max_new_marker_error"  type="double" value="$(arg max_new_marker_error)" />
		<param name="max_track_error"       type="double" value="$(arg max_track_error)" />
		<param name="output_frame"          type="string" value="$(arg output_frame)" />

		<remap from="camera_image"  to="$(arg cam_image_topic)" />
		<remap from="camera_info"   to="$(arg cam_info_topic)" />
		<param name="max_frequency"       type="double" value="15" />
	</node>
</launch>
